### DISCUSSION TITLE
Audit Log Required Permissions

### DISCUSSION THREAD

**Discussion by grantmcd** [community/community#11746] [Other Feature Feedback, Questions, & Ideas]
It seems strange to me that to view Audit Logs from the API for an organization, the user access token requires full `admin:org` permissions. 

My organization has use a use case where we'd like to consume the Audit Logs and ingest them into other security platforms, but to require the UAT to have full admin write access to the organization is scary.

I think having another permission specifically for Audit Logs or allowing `read:org` to access them would be very helpful!

**Comment by amitz-val** [comment]
bump, its been over a year.
Github Staff - you must give an explanation, this discrepancy is actually pretty common with other endpoints, the graphql api is really unmaintained and does not have an appropriate docs.

**Reply by amitz-val** [reply]
making it clearer - I stumbled upon something similar, that with the same authorized App token, i could access auditlogs via the REST api but not via graphql.

**Reply by amitz-val** [reply]
Not to mention the absolute lack of information about the failure or what scopes are required to do the request in graphql.

**Reply by Jmainguy** [reply]
I appreciate you very much, could not figure out why it said I didnt have permission, when I clearly gave my user auditLog permission. adding admin:org did the trick
